### ISSUE TITLE
0.3.0: crash on reflow

### ISSUE THREAD

**Issue by ninjalj** [contour-terminal/contour#595]
### Contour Terminal version

Contour Terminal Emulator 0.3.0-unreleased-v0.2.3.182-160-g21c4dab7-21c4dab7

### Installer source

Github: source code cloned

### Operating System

Linux

### Architecture

x86-64

### Steps to reproduce

- for i in {1..200}; do echo $i foobarbazquxquux foo bar baz qux quux; done
- Resize the window to near-zero size.

### Actual Behavior

```
[/home/lj/src/term/contour/src/terminal/Grid.cpp:155] Precondition failed. totalLineCount() >= linesUsed_

========================================================================
  An internal error caused the terminal to crash ;-( 😭
-------------------------------------------------------

Please report this to https://github.com/contour-terminal/contour/issues/

Contour version: 0.3.0-unreleased-v0.2.3.182-160-g21c4dab7-21c4dab7

Stack Trace:
------------
/home/lj/src/term/contour/src/crispy/StackTrace.cpp:90
/home/lj/src/term/contour/src/crispy/StackTrace.cpp:218
/home/lj/src/term/contour/src/contour/ContourApp.cpp:105
/home/lj/src/term/contour/src/contour/ContourApp.cpp:119
0x7f7421827bd0
0x7f7421827b24
0x7f7421811557
/home/lj/src/term/contour/src/crispy/assert.h:54
/home/lj/src/term/contour/src/crispy/assert.h:61
/home/lj/src/term/contour/src/terminal/Grid.cpp:155
/home/lj/src/term/contour/src/terminal/Grid.cpp:885
/home/lj/src/term/contour/src/terminal/Grid.cpp:897
/home/lj/src/term/contour/src/terminal/Screen.cpp:490
/home/lj/src/term/contour/src/terminal/Terminal.cpp:799
/home/lj/src/term/contour/src/contour/helper.cpp:478
/home/lj/src/term/contour/src/contour/opengl/TerminalWidget.cpp:406
0x7f7422cad9e1
0x7f7422c8e1bc
/home/lj/src/term/contour/src/contour/opengl/TerminalWidget.cpp:550
0x7f7422c49bbf
0x7f7421fad1f8
0x7f7422c84232
0x7f7422c8487e
/usr/include/qt5/QtWidgets/qwidget.h:854
/home/lj/src/term/contour/src/contour/ScrollableDisplay.cpp:64
0x7f7422c8e1bc
0x7f7422c49bbf
0x7f7421fad1f8
0x7f7422c84232
0x7f7422c8513b
0x7f7422c8f67b
/home/lj/src/term/contour/build/src/contour/contour_autogen/EWIEGA46WW/moc_ScrollableDisplay.cpp:109
0x7f7421de5c3a
0x7f7421de22b0
0x7f7421de6371
0x7f7421de034a
0x7f7422d41a45
0x7f7422d80b58
0x7f7422db1765
0x7f7422db71bd
0x7f7422c69d18
0x7f7422c49ba3
0x7f7421fad1f8
0x7f7422ca6fa6
0x7f7422caaf70
0x7f7422c49bbf
0x7f7421fad1f8
0x7f74223af907
0x7f74223919a3
0x7f741e5f051a
0x7f742131bfcb
0x7f742131c265
0x7f742131c32f
0x7f7421ffff68
0x7f7421fabae3
0x7f7421fb418d
/home/lj/src/term/contour/src/contour/ContourGuiApp.cpp:330
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/include/g++-v11/bits/invoke.h:75 (discriminator 4)
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/include/g++-v11/bits/invoke.h:98
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/include/g++-v11/functional:423
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/include/g++-v11/functional:506
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/include/g++-v11/bits/invoke.h:61
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/include/g++-v11/bits/invoke.h:117
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/include/g++-v11/bits/std_function.h:293
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/include/g++-v11/bits/std_function.h:561
/home/lj/src/term/contour/src/crispy/App.cpp:234
/home/lj/src/term/contour/src/contour/ContourGuiApp.cpp:67
/home/lj/src/term/contour/src/contour/main.cpp:24
0x7f742181282e
??:?
========================================================================

```

**Comment by Utkarsh-khambra** [comment]
I can't reproduce this issue. Is it still happening to you?

**Comment by ninjalj** [comment]
Duplicate of #610. Fixed by #628

**Comment by christianparpart** [comment]
Thanks for the feedback @ninjalj :)
